package.json

```
@@ -54,7 +54,7 @@
     ],
     "dependencies": {
         "@hicaru/bearby.js": "^0.5.6",
-        "@massalabs/web3-utils": "^1.4.8",
+        "@massalabs/web3-utils": "^1.4.9-dev.20240301101610",
```

```suggestion
        "@massalabs/web3-utils": "^1.4.9-dev",
```

and run npm install after to update the package-lock.json